app/routers/api/files.py

```
@@ -128,7 +128,137 @@ def _serialize_file(
+                and_(sort_field.is_(None), FileInventory.id > last_id),
+            )
+        )
+    return query.filter(
+        or_(
+            sort_field > last_sort_value,
+            and_(sort_field == last_sort_value, FileInventory.id > last_id),
+        )
+    )
+
+
+def _generate_next_cursor(files_list, sort_by, valid_sort_fields):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_generate_next_cursor`

(ANN202)